### ISSUE TITLE
Client Rates and Discounts

### ISSUE THREAD

**Issue by markwilliamfirth** [dwyl/hq#325]
@iteles I noticed with healthlocker new budget vs old budget (https://github.com/dwyl/hq/issues/318) the rates changed but the discount stayed the same

It might be a good idea for us to assign fixed rates (based on the market rate) to roles and then apply client discounts as appropriate. In this case we could have lifted the discounts rather than increasing the rates.

**Comment by iteles** [comment]
@markwilliamfirth This was a very particular case to the agreement we have with Healthlocker, there was only one rate that increased and that was the one rate we were making a _loss_ on. Now we're able to cover costs but the discount needed to stay as per the original agreement.

**Comment by markwilliamfirth** [comment]
@iteles Understood it was a particular case

Going forward it might be more ideal to set our rates against the market and provide discounts for preferred clients as appropriate. The next step is to do research into market rates - has any been done before?

**Comment by nelsonic** [comment]
@markwilliamfirth the "_market_" rate depends entirely on what "_market_" we are _in_.
Are we a _high_ quality professional services team?
or a technology "outsourcing" company?
The end-result to the client might be _similar_
But the rate will vary by 300-500% ... 
It _all_ depends on what the medium-to-long-term _plan_ is for Services. 🤔

**Comment by iteles** [comment]
Closing this as the question has been answered - rates are not fixed across 'markets' and therefore not across all our clients. 
To be expanded on in https://github.com/dwyl/process-handbook/issues/45 and an issue on researching market rates can be reopened later this year if we confirm what we need to research.

**Comment by markwilliamfirth** [comment]
I think before we can close this we can establish which markets we are targeting

e.g. http://www.eyecatchermc.com/

these guys are a branding agency that work exclusively with startups in the health tech space

If we can narrow down some of our markets
e.g. development services for startups in the healthtech space then we can look at market rates and answer this issue

Definition of markets depends on #328

**Comment by iteles** [comment]
@markwilliamfirth Did you read my comment above about the question in _this_ issue being closed and a new issue is to be opened when we make that exact determination?

**Comment by markwilliamfirth** [comment]
I think it's fine to leave this open and have a blocker on it until 328 is done

**Comment by iteles** [comment]
Really wanted to use  this gif but couldn't get it to display! https://img-s3.onedio.com/id-57e132d5950fdb64063725bc/rev-0/raw/s-d573ece8748dd63218413b37554acae8a21eabf8.gif

**Comment by nelsonic** [comment]
![child-candle](https://user-images.githubusercontent.com/194400/52019855-087d8580-24e7-11e9-9cb6-0e699bb6c012.gif)
